ValidationLibrary.Rules.Tests/HasLicenseRuleTests.cs

```diff
@@ -53,7 +40,7 @@ public async Task IsValid_ReturnsInvalidForPublicRepositoryWithLicense()
         {
```

Comment:
Is line 39 this supposed to be *WithoutLicence*  at the end?

Reply:
No, as said above.

Reply:
Sorry, misread, yes, it's supposed  to be without.